### PR TITLE
Caitlin G's Testing

### PR DESCRIPTION
**Pull request by caitlin-g**
@bradiowave

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,478 @@
...
+
+    const actual = helpers.multiplyByTen(10);
+    expect(actual).toEqual(expected);
+  });
+
+  it("should throw an error if a string is passed", () => {
+    expect(() => {
+      helpers.multiplyByTen("123");
+    }).toThrow();
+    expect;
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:24]: Good test for data validation.

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,478 @@
...
+    const actual = helpers.lessThanNinety(80);
+    expect(actual).toEqual(expected);
+  });
+
+  it("should check if values are not less than 90", () => {
+    const expected = false;
+
+    const actual = helpers.lessThanNinety(91);
+    expect(actual).toEqual(expected);
+  });
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:118]: Good job testing right where the expected output will change.

File: basic-javascript/project-2.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 /* eslint-disable arrow-parens */
 
 const getBiggest = (x, y) => {
+  if (!x || !y) {
+    return null;
+  }
+  if (typeof x === "string" || typeof y === "string") {
+    throw new Error("value must be a number");
+  }
```
Comment (bradencodes) [inline basic-javascript/project-2.js:9]: Great job adding data checking to the functions, and testing for them.

**Comment by bradencodes** [comment]
## MVP
- [x] Write tests for project-1.js
- [x] Write tests for project-2.js

## Stretch
- [x] Correct bugs and refactor the code
- [ ] Write tests for arrays.js
- [ ] Write tests for object.js

## What Looks Great
- MVP complete! Nice work
- Frequent and descriptive commits
- Code is well formatted and easy to read
- Titles for tests are descriptive
- Tests are thorough and check for edge cases, true outputs, and false outputs

## What Could Be Added
- Try your luck with the stretch problems for extra practice with testing!

## Rating (1-3)
**3**
